pkg/storage/storage.go

```diff
@@ -29,6 +29,11 @@ import (
 type Storage struct {
 	driver.Driver
 
+	// MaxHistory specifies the maximum number of historical releases that will
+	// be retained, including the most recent release. Values of 0 or less are
```

Comment:
- [ ] Ah, here we note what we do with negative values. If you decide to stick with a signed integral type, consider adapting this documentation to the other structures where we store this same kind of value. I don't think we'd make it this far with a negative value, but I agree that it's better to make each component safe, not trusting that an outer layer will have validated the input beforehand.